.travis.yml

```diff
@@ -6,6 +6,7 @@ cache:
 
 os:
   - linux
+  - macos
```

Comment:
I think you need to specify `osx` instead of `macos`.

https://docs.travis-ci.com/user/reference/osx/#using-macos

Reply:
doh, :sweat_smile: